data/p/part-2-how-to-open-a-pull-request.mdx

```diff
@@ -1,6 +1,7 @@
 ---
 title: "How to open a pull request"
 date: "2021-11-03"
+lastmod: "2025-04-03"
 tags: ["pull request guide"]
 draft: false
 summary: "In Part II of our Pull Request Guide, we will dive into the processes of opening to merging a pull request and the best advices and integrations to master your code reviews."
```

Comment:
je crois que leur reco a un summary un peu différent

Reply:
In Part II of our Pull Request Guide, we will dive into the processes of opening to merging a pull request and the best recommendations and integrations to master your code reviews.

Reply:
merci [](edited-with-Axolo)